packages/website/ts/pages/jobs/benefits.tsx

```diff
@@ -1,109 +1,134 @@
+    'Meals and snacks provided in-office daily.',
+];
+
+interface Value {
+    iconSrc: string;
+    text: string;
+}
+const VALUES: Value[] = [
     {
-        bulletColor: '#EB5757',
-        description:
-            'Donec eget auctor mauris, a imperdiet ante. Ut a tellus ullamcorper, pharetra nibh sed, dignissim mauris. Quisque vel magna vitae nisi scelerisque commodo sed eget dolor. Maecenas vehicula orci',
+        iconSrc: 'images/jobs/heart-icon.svg',
+        text: 'Do the right thing',
     },
     {
-        bulletColor: '#6FCF97',
-        description:
-            'Donec eget auctor mauris, a imperdiet ante. Ut a tellus ullamcorper, pharetra nibh sed, dignissim mauris. Quisque vel magna vitae nisi scelerisque commodo sed eget dolor. Maecenas vehicula orci',
+        iconSrc: 'images/jobs/ship-icon.svg',
+        text: 'Consistently ship',
     },
     {
-        bulletColor: '#56CCF2',
-        description:
-            'Donec eget auctor mauris, a imperdiet ante. Ut a tellus ullamcorper, pharetra nibh sed, dignissim mauris. Quisque vel magna vitae nisi scelerisque commodo sed eget dolor. Maecenas vehicula orci',
+        iconSrc: 'images/jobs/calendar-icon.svg',
+        text: 'Focus on long term impact',
     },
 ];
-const LARGE_LAYOUT_HEIGHT = 937;
-const LARGE_LAYOUT_BENEFITS_LIST_PADDING_LEFT = 205;
-const HEADER_TEXT = 'Benefits';
-const BENEFIT_ITEM_MIN_HEIGHT = 150;
 
 export interface BenefitsProps {
     screenWidth: ScreenWidths;
 }
 
-export const Benefits = (props: BenefitsProps) => (
-    <div style={{ backgroundColor: colors.jobsPageBackground }}>
-        {props.screenWidth === ScreenWidths.Sm ? <SmallLayout /> : <LargeLayout />}
-    </div>
-);
-
-const LargeLayout = () => (
-    <div className="flex" style={{ height: LARGE_LAYOUT_HEIGHT }}>
-        <div style={{ width: '43%', height: '100%' }}>
-            <ImageGrid />
-        </div>
-        <div
-            className="pr4"
-            style={{ paddingLeft: LARGE_LAYOUT_BENEFITS_LIST_PADDING_LEFT, width: '57%', height: '100%' }}
-        >
-            <BenefitsList />
-        </div>
-    </div>
-);
+export const Benefits = (props: BenefitsProps) => {
+    const isSmallScreen = props.screenWidth === ScreenWidths.Sm;
+    return (
+        <Container className="flex flex-column items-center py4 sm-px3" backgroundColor={colors.white}>
+            {!isSmallScreen ? (
+                <Container className="flex" maxWidth="800px">
+                    <BenefitsList />
+                    <Container marginLeft="215px">
+                        <ValuesList />
+                    </Container>
+                </Container>
+            ) : (
+                <Container className="flex-column">
+                    <BenefitsList />
+                    <Container marginTop="50px">
+                        <ValuesList />
+                    </Container>
+                </Container>
+            )}
+        </Container>
+    );
```

Comment:
using the media queries abstraction tool seems better suited here.

What I've also seen done is expand the `Container` API to also take in an object for each (relevant) prop. For example:
```
<Container maxWidth={{
  small: 800,
}}>
... 
</Container>
```
Again, just a thought. Careers page is pretty isolated, so whatever works is good.

Reply:
yea lets just leave it